website/docs/r/is_subnet.html.markdown

```diff
@@ -50,7 +50,12 @@ The `ibm_is_subnet` resource provides the following [Timeouts](https://www.terra
 ## Argument reference
 Review the argument references that you can specify for your resource. 
 
-- `access_tags`  - (Optional, List of Strings) A list of access management tags to attach to the subnet. **Note** For more information, about creating access tags, see [working with tags](https://cloud.ibm.com/docs/account?topic=account-tag).
+- `access_tags`  - (Optional, List of Strings) A list of access management tags to attach to the subnet.
+  **Note:**
+  - For more information, about creating access tags, see [working with tags](https://cloud.ibm.com/docs/account?topic=account-access).
+  - To create an `access` tag, you must have the access listed in the [Granting users access to tag resources](https://cloud.ibm.com/docs/account?topic=account-access)
+  - `access_tags` must be in the form `key:value`.
```

Comment:
Should we add another note -  create access tag using "ibm_resource_tag" resource. You can attach only access tags that already exists.

Reply:
Sure, adding that